pkg/infra/game/stage/fight/fight.go

```diff
@@ -13,6 +13,9 @@ import (
 )
 
 func DealDamage(damageToDeal uint, agentsFighting []string, agentMap map[commons.ID]agent.Agent, globalState *state.State) {
+	if len(agentsFighting) == 0 {
+		return
+	}
```

Comment:
Is this correct?

Reply:
Yeah I got a divide by 0 error once as the damage is divided by the number of agents so just added this check to make sure that never happens

Reply:
This is super strange now then since we now check for if all agents cower